src/components/Fibonacci.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-mixed-spaces-and-tabs */
+import React, {useMemo, useState} from 'react';
+
+export const Fibonacci = () => {
```

Comment:
make folder name 
ComponentFibonacci 
and add 
ComponentFibonacci.tsx
file name should be same as function name